Content modifications

robertsky opened the pull request:
Connect:
Added more details to Facebook section.
Reworded the news organisation part to be more friendly and to alert us for new articles.

Membership:
Added minor detail on the locker nametags.

kaihendry commented:
Don't like the fact that a news organisation has to email HSG to publish an article. Extra bit of bureaucracy that could put people off. We can find out what people say about HSG using https://www.google.com/alerts

Also I feel we should encourage people to join a mailing list instead of Facebook. Since it's expensive / difficult to reach all members via the centralised closed Facebook platform.

hyperair commented:
I think it should be made optional. Something like: "An email to blah@hackerspace.sg alerting us would be nice, but is not mandatory."

Valentine commented:
What @hyperair said. Also, Fair Use is not a licence, and it's likely that some news organisations (however loosely defined) cannot use images with a Creative Commons licence. It's only images, right?

rolandturner commented:
I don't want to get into whether or not to make this specific change, but:

  * Fair use rights are intrinsic, they are not related to permission.
    We don't need to mention them.
  * The Creative Commons language proposed does not identify a specific
    license, and therefore does not make clear what is being granted.
    There's a tool on the CC website for making this decision and
    generating appropriate language. I'd suggest using it; and choosing
    the most permissive option, of course.
  * News organisations don't have a blanket problem with CC licenses.
    The non-commercial and non-derivative constraints are likely to be a
    problem, but basic attribution has been normal news media practice
    for decades.

- Roland



On 04/06/2015 03:47 PM, Valentine C wrote:
>
> What @hyperair <https://github.com/hyperair> said. Also, Fair Use is 
> not a licence, and it's likely that some news organisations (however 
> loosely defined) cannot use images with a Creative Commons licence. 
> It's only images, right?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/hackerspacesg/hackerspace.sg/pull/119#issuecomment-89962021>.
>

robertsky commented:
@kaihendry: Your dislike for Facebook does not hide the fact that it is
there. What's considered expensive/difficult to reach to you might be cheap
and easy for the others and vice versa. I had considered your position
while editing the Facebook portion, hence 'an alternative". I have also
adjusted the request to alert hackerspace of articles to be less formal,
that of @hyperair's.

@Valentine, @rolandturner: I was puzzled by the previous commit actually,
hence the edit. It was done in a rush, hence making it more muddled. I have
made further edits to remove Fair Use and Creative Commons and made
reference to a set of principles to guide journalists in evaluating what's
consider Fair Use. I have adjusted.

On 6 April 2015 at 15:59, Roland Turner <notifications@github.com> wrote:

> I don't want to get into whether or not to make this specific change, but:
>
> * Fair use rights are intrinsic, they are not related to permission.
> We don't need to mention them.
> * The Creative Commons language proposed does not identify a specific
> license, and therefore does not make clear what is being granted.
> There's a tool on the CC website for making this decision and
> generating appropriate language. I'd suggest using it; and choosing
> the most permissive option, of course.
> * News organisations don't have a blanket problem with CC licenses.
> The non-commercial and non-derivative constraints are likely to be a
> problem, but basic attribution has been normal news media practice
> for decades.
>
> - Roland
>
>
>
> On 04/06/2015 03:47 PM, Valentine C wrote:
> >
> > What @hyperair <https://github.com/hyperair> said. Also, Fair Use is
> > not a licence, and it's likely that some news organisations (however
> > loosely defined) cannot use images with a Creative Commons licence.
> > It's only images, right?
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/hackerspacesg/hackerspace.sg/pull/119#issuecomment-89962021
> >.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackerspacesg/hackerspace.sg/pull/119#issuecomment-89965180>
> .
>

robertsky commented:
Hello?7 months and pending. Any feedback? If not, I would like to have this merged in.

valentine commented:
Think you need to rebase and cherry pick.

robertsky commented:
@valentine opps. just spotted this. should have done this on a branch instead. anyway, have resolved the issues.
